data-model/fdo-type/source-system/0.3.0/schema/source-system.json

```diff
@@ -8,7 +8,7 @@
     "@id": {
       "type": "string",
       "description": "The unique identifier (handle) of the Source System object",
-      "pattern": "^https:\/\/hdl\\.handle\\.net\/[\\w.]+\/(.){3}-(.){3}-(.){3}",
+      "pattern": "^https://hdl\\.handle\\.net/[\\w.]+/(.){3}-(.){3}-(.){3}",
```

Comment:
Was this a mistake? I think you need to escape these `/`

Also, if you're changing the pattern, i think it should be `\w+` not `[\\w.]+`

Reply:
Reverted regex. I tested but you need the `.` in there to allow dots in the prefix part. Tested with https://regexr.com/ and this should be alright.

Reply:
Right right, looks good